packs/data/equipment.db/oil-of-unlife-greater.json

```diff
@@ -31,7 +31,7 @@
         },
         "containerId": null,
         "description": {
-            "value": "<p><strong>Activate</strong> <span class=\"pf2-icon\">1</span> Interact</p>\n<p>This thin, black liquid imparts a bone-deep chill while rapidly repairing an undead creature's physical or spiritual form. When you dash oil of unlife onto an undead creature, or a living creature with negative healing, the oil absorbs quickly into its body, and the creature regains 6d8+20 Hit Points. You can pour oil of unlife on an incorporeal undead; in this case, the creature absorbs the oil into itself.</p>"
+            "value": "<p><strong>Activate</strong> <span class=\"pf2-icon\">1</span> Interact</p>\n<p>This thin, black liquid imparts a bone-deep chill while rapidly repairing an undead creature's physical or spiritual form. When you dash oil of unlife onto an undead creature, or a living creature with negative healing, the oil absorbs quickly into its body, and the creature regains [[/r {6d8+20}[healing]]]{6d8+20 Hit Points}. You can pour oil of unlife on an incorporeal undead; in this case, the creature absorbs the oil into itself.</p>"
```

Comment:
The issue is referring to the on consume field on the details tab, which makes a roll when the item is consumed. So it would need to be added there too.

Reply:
Good call!